### ISSUE TITLE
Devtool not making the connection

### ISSUE THREAD

**Issue by BayBreezy** [arashsheyda/nuxt-mongoose#16]
Hello 👋,

Thank you for creating this module. I like what it offers.
I am trying to use the devtools to create api endpoints(really love that feature) but it is not working at the moment. In my VSCode terminal, I get no errors when the application starts up. But in the browser console, I am seeing a few warnings/ errors.

The Errors from the browser are:

- GET http://localhost:3000/@vite/client net::ERR_ABORTED 404 (Page not found: /@vite/client)
- rpc.50d424f0.js:1 [vite-plugin-hot] Failed to import "/@vite/client"
- Uncaught (in promise) Error: Unable to connect to devtools
    at De (rpc.50d424f0.js:1:9272)

Here are some screenshots that may help:

#### Browser:

<img width="1382" alt="image" src="https://github.com/arashsheyda/nuxt-mongoose/assets/53905713/7743b597-dde5-4010-a70a-984f6a5da944">

#### Terminal:

<img width="844" alt="image" src="https://github.com/arashsheyda/nuxt-mongoose/assets/53905713/9b82cd6e-6f44-4347-9d9a-67fcd5331c37">

**Comment by arashsheyda** [comment]
@BayBreezy thanks for pointing this out, yes it's due to DevTools changes, it is from the `rpc client` and the fix is easy, but I want to refactore some thing and I havn't got the time yet :)

**Comment by BayBreezy** [comment]
@arashsheyda Ok. I understand. I will wait on the refactor. 
Thanks

**Comment by BayBreezy** [comment]
Hello @arashsheyda , just following here. 
Wondering if you can provide an ETA on when this will be updated?

**Comment by arashsheyda** [comment]
sorry for the long wait! I did refactor that part long time ago, though there is another issue that I'm trying to fix(I don't know why but after updating dependencies it throw an error if your project doen't have mongoose as dependency!) and I think it will be done in couple of days. thank you for your wait!

**Comment by BayBreezy** [comment]
Ok, I will. Thanks for the update. I actually saw another bug yesterday where after running the posting tall script, the code would hang. I will test to see if this is resolved with the update as well. 

The hang would take place when I try to connect to a momgodb that is secured, like atlas
